src/QuadForm/Herm/Lattices.jl

```diff
@@ -750,3 +750,114 @@ function maximal_integral_lattice(V::HermSpace)
   return maximal_integral_lattice(L)
 end
 
+################################################################################
+#
+#  Intersection
+#
+################################################################################
+
+#=
+    _reconstruction_herm_lattice(L::ZLat, E::Hecke.NfRel{nf_elem}, 
```

Comment:
For unofficial comments, like internal functions, you don't need this part, only the comments on what it is doing from what